src/main/frontend/ui.cljs

```diff
@@ -994,7 +994,7 @@
   [state visible? content-fn ref]
   [:div.lazy-visibility
    {:ref ref
-    :style {:min-height 24}}
+    :style {:min-height 0}}
```

Comment:
@YU000jp Thank you for your contribution!
The default value of `min-height` is zero, so you can completely remove the style attribute.

Reply:
@tiensonqin I removed the style attribute. `lazy-loading-placeholder` has its own height property set, so it will be rendered properly. I am not sure if there is any other reason to keep the hardcoded min-height.

Reply:
It's introduced in
https://github.com/logseq/logseq/commit/92adab37f99e4b58107bc60735bf66952d548b89
And previously it looks https://github.com/logseq/logseq/commit/92adab37f99e4b58107bc60735bf66952d548b89#diff-b4cc60fce69f679ffdd20aab6324bbb260cf74dbee8a08e539aabefbd1d895eaL916-L918